tests/testthat/test-tbl_ard_summary.R

```diff
@@ -192,3 +192,19 @@ test_that("tbl_ard_summary(statistic) error messages", {
       tbl_ard_summary(by = ARM, statistic = list(AGE = c("{mean}", "{median}")))
   )
 })
+
+# regression test for #1765 ----------------------------------------------------
+test_that("tbl_ard_summary() produces warnings when glue selects first stat from a group", {
+
+  trial1 <- head(trial)
+  stack_ard <- ard_stack(
```

Comment:
this ard stack has both continuous and categorical but then we require categorical stat from the numeric age hence the issue

Reply:
the brdg_summary example is clearer imo